src/sagas/modelSaga.js

```diff
@@ -1,15 +1,17 @@
 import { takeLatest, put, select } from 'redux-saga/effects';
 import { ModelManager } from 'composer-concerto';
-import * as selectors from '../selectors/modelSelectors';
+import * as clauseTemplateSelectors from '../selectors/clauseTemplateSelectors';
 import * as actions from '../actions/modelActions';
+import * as clauseTemplateActions from '../actions/clauseTemplatesActions';
 
 /**
  * saga which yields to putting the successful model manager into the store
  * and subsequently clears all model manager errors from the store
  */
-export function* validateModelFiles() {
-  // get all the model files in the state
-  const modelFiles = yield select(selectors.modelFiles);
+export function* validateClauseModelFiles(action) {
+  // get all the model files for a template
```

Comment:
It could be helpful to provide more step-by-step comments to describe what is happening in the sagas.

Reply:
this particular one seems well commented / easy to understand to me. Is there a specific spot you think could use a comment? I will add comments to some of the other more complicated sagas that could use them.

Reply:
It feels like it makes less sense now than when I wrote this note, but I had in mind that it's not readily apparent what is happening with modelFiles and modelManagers if someone is not quite familiar with cicero. Another example could be `addToContract` in `src/sagas/contractSaga.js`, where someone may not be very familiar with Slate. But maybe this is overkill.

Reply:
Yes, I agree with `addToContract`. I just added comments to that one in my local branch. I'll add a comment for the modelManager line to this one here as well.